pkg/cmd/root/help_topic.go

```diff
@@ -104,22 +104,22 @@ var HelpTopics = map[string]map[string]string{
 			- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
 			- %[1]stimefmt <format> <time>%[1]s: formats a timestamp using Go's Time.Format function
 			- %[1]struncate <length> <input>%[1]s: ensures input fits within length
-
-			EXAMPLES
-			  # format issues as table
-			  $ gh issue list --json number,title --template \
-			    '{{range .}}{{tablerow (printf "#%%v" .number | autocolor "green") .title}}{{end}}'
-			
-			  # format a pull request using multiple tables with headers
-			  $ gh pr view 3519 --json number,title,body,reviews,assignees --template \
-			    '{{printf "#%%v" .number}} {{.title}}
-
-			    {{.body}}
-
-			    {{tablerow "ASSIGNEE" "NAME"}}{{range .assignees}}{{tablerow .login .name}}{{end}}{{tablerender}}
-			    {{tablerow "REVIEWER" "STATE" "COMMENT"}}{{range .reviews}}{{tablerow .author.login .state .body}}{{end}}
-			    '
 		`, "`"),
+		"example": heredoc.Doc(`
+			# format issues as table
+			$ gh issue list --json number,title --template \
+		  		'{{range .}}{{tablerow (printf "#%%v" .number | autocolor "green") .title}}{{end}}'
```

Comment:
Double `%%` should now be a single `%`. Double `%%` was previously necessary because it was in a Docf (sprintf-like method), but now it's a regular Doc method.

Reply:
Fixed.